src/pages/Login/component.js

```diff
@@ -36,6 +43,7 @@ const Login = ({ handleSubmit }) => (
               />
               <Form.Item>
                 <Button
+                  loading={isSubmitting}
```

Comment:
To avoid multiple requests to server it's a good practice to disable button during submit form, just use the same `isSubmitting` prop

Reply:
Fixed